framework/auth/views.py

```diff
@@ -80,10 +80,10 @@ def forgot_password_post():
         if user_obj:
             #TODO: Remove this rate limiting and replace it with something that doesn't write to the User model
             now = datetime.datetime.utcnow()
-            last_attempt = user_obj.forgot_password_last_post or now - datetime.timedelta(seconds=FORGOT_PASSWORD_MINIMUM_TIME)
+            last_attempt = user_obj.forgot_password_last_post or now - datetime.timedelta(seconds=settings.FORGOT_PASSWORD_MINIMUM_TIME)
```

Comment:
I'll change this if absolutely necessary- but as you say, merge conflicts are going to be a pain point for this PR no matter what.

My strong preference would be to resolve this by giving the developer a heads-up , rather than hacking at the code. The developer can (and should) be trusted to examine the final diff for a merge conflict before moving to review.